### PR TITLE
Rename Python package

### PR DESCRIPTION
**Pull request by arsenius7**
* 'sonata' -> 'libsonata'

### CODE REVIEW

**Comment by wizmer** [comment]
Do we already have packages depending on this one ?

**Comment by arsenius7** [comment]
Up to my knowledge, only `bluepy`; and once we have the wheels at PyPI, the transition would be trivial.

**Comment by arsenius7** [comment]
I think I did at `python/CMakeLists.txt`. Or is there some other place I already forgot? ^^

File: python/sonata/__init__.py.in [hunk 1]
```diff
@@ -1,3 +1,3 @@
-from ._sonata import *
```
Comment (wizmer) [inline python/sonata/__init__.py.in]: I meant that the folder in which this file exists should be renamed, no ?

Comment (arsenius7) [reply python/sonata/__init__.py.in]: OK, thanks.

It did not affect the build, but I have renamed the source folder nevertheless for the sake of consistency.

**Pull request merged**